README.md

```diff
@@ -85,6 +85,12 @@ Integration tests can be run on any deployed DC/OS cluster. For installation ins
 
 Integration tests are installed via the [dcos-integration-test](./packages/dcos-integration-test/) Pkgpanda package.
 
+Integration test files are stored on the DC/OS master node at `/opt/mesosphere/active/dcos-integration-test`.
+Therefore, in order to test changes to test files, move files from `packages/dcos-integration-test/extra/` in your checkout to `/opt/mesosphere/active/dcos-integration-test` on the master node.
+
+The canonical source of the test suite's results is the continuous integration system.
```

Comment:
You may also want to have a look at TeamCity's configuration - some tests are muted, some are disabled for particular builds. I think that we should be able to filter out at least some of the flaky tests this way.